config/crd/bases/osp-director.openstack.org_openstackvmsets.yaml

```diff
@@ -35,7 +35,7 @@ spec:
         metadata:
           type: object
         spec:
-          description: VMSetSpec defines the desired state of VMSet
+          description: OpenStackVMSetSpec defines the desired state of VMSet
```

Comment:
Should `VMSet` here and below be renamed?

Reply:
Yes. I will fix this (in the types.go file..) and regenerate the CRD